examples/cpu/plotting.cpp

```diff
@@ -47,40 +55,38 @@ int main(void){
      * it can be used for rendering text */
     fg::Font fnt;
 #ifdef OS_WIN
-    fnt.loadSystemFont("Calibri", 32);
+    fnt.loadSystemFont("Calibri");
 #else
-    fnt.loadSystemFont("Vera", 32);
+    fnt.loadSystemFont("Vera");
```

Comment:
Sure, we can use a default font setup internally that is replaced if user explicitly asks for it.